fe/fe-core/src/main/java/com/starrocks/qe/scheduler/QueryRuntimeProfile.java

```diff
@@ -71,11 +71,13 @@ public class QueryRuntimeProfile {
      * Set the queue size to a large value. The decision to execute the profile process task asynchronously
      * occurs when a listener is added to {@link QueryRuntimeProfile#profileDoneSignal}. The function
      * {@link QueryRuntimeProfile#addListener} will then determine if the size of the queued task exceeds
-     * {@link Config#profile_process_blocking_queue_size}.
+     * queue size.
      */
     private static final ThreadPoolExecutor EXECUTOR =
-            ThreadPoolManager.newDaemonFixedThreadPool(Config.profile_process_threads_num,
-                    Integer.MAX_VALUE, "profile-worker", false);
+            ThreadPoolManager.newDaemonCacheThreadPool(
```

Comment:
so thread pool core size is 3/4 * all cpus, maximumPoolSize is 3 * All cpus? can you explain  why you decide using these numbers

Reply:
no. 
- `corePoolSize`: 0
- `maxPoolSize`: 3/4 * CPUCores
- `queueSize`: 3 * CPUCores